src/main/java/com/launchacademy/giantleap/models/Review.java

```diff
@@ -0,0 +1,42 @@
+package com.launchacademy.giantleap.models;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "reviews")
+@Setter
+@Getter
+@NoArgsConstructor
+public class Review {
+
+  @Id
+  @SequenceGenerator(name="review_generator", sequenceName="reviews_id_seq", allocationSize = 1)
+  @GeneratedValue(strategy= GenerationType.SEQUENCE, generator="review_generator")
+  @Column(name="id", nullable=false, unique=true)
+  private Integer id;
+
+  @ManyToOne(cascade = CascadeType.MERGE)
+  @JoinColumn(name = "character_id")
+  private Integer characterId;
+
```

Comment:
don't forget you'll need to set up the @OneToMany on the characters table to complete the link